test/test_util/react_util.dart

```diff
@@ -287,7 +287,7 @@ bool _hasTestId(Map props, String key, String value) {
 
   bool first = false;
 
-  var results = react_test_utils.findAllInRenderedTree(root, allowInterop((descendant) {
+  var results = react_test_utils.findAllInRenderedTree(root, allowInterop((Function descendant) {
```

Comment:
Pretty sure `descendant` is always either `ReactComponent` or `Element`, not `Function`...